### DISCUSSION TITLE
Consulta sobre wireframe

### DISCUSSION THREAD

**Discussion by Darkonis13** [dds-jv/foro#325] [TPA - 2021] [answered]
Buenas quería consultar sobre cómo quieren el maquetado para el 05/11, ¿Es una presentación con imágenes de cómo se vería la página, o además tiene que tener algún comportamiento?

Muchas gracias :)

**Comment by gastonprieto** [comment, marked as answer]
Buenas @Darkonis13 

Serian imagenes prototipo de tus pantallas, con a lo suma flechas o algunos items para marcar hacia que otra pantalla van.
